src/content/admin/previews.mdx

```diff
@@ -0,0 +1,65 @@
+description: Manage preview environments from the Admin Dashboard
+sidebar_label: Preview Environments
+id: previews
+---
+
+import TiersList from "@theme/TiersList";
+
+# Managing Preview Environments <TiersList tiers="Scale Enterprise Self-Hosted" />
+
+The Previews section of the Admin Dashboard provides administrators with a centralized view to manage all [Preview Environments](../previews/index.mdx) in your Okteto instance.
+
+
+## Preview Scope
+
+Preview environments can have one of two scopes:
+
+- **Global Scope**: The preview environment is accessible to all members of your organization. These previews are visible to the entire team and can be managed by users with appropriate permissions.
+- **Personal Scope**: The preview environment is only accessible to the owner. These previews are indicated with a user icon next to the preview name in the preview list and are private to that account.
+
+The scope determines who can view and interact with the preview environment within your organization.
+
+## Filtering and Searching Previews
+
+The preview environments list provides several filtering options to help you find specific previews:
+
+- **Search** - Search for previews by name using the search box
+- **Repository** - Filter by one or more repositories (searchable dropdown)
+- **Status** - Filter by preview status such as active, sleeping, or error states
+- **Owner** - Filter by the user who created the preview (searchable dropdown)
+- **Updated** - Filter by when the preview was last updated:
+  - Last hour
+  - Last 24 hours
+  - Last 7 days
+  - Last 30 days
+  - Last 90 days
+
+You can combine multiple filters to narrow down your search results.
+
+## Preview Table Columns
+
+The preview environments table displays the following information for each preview:
+
+- **Name** - The preview environment name with a user icon indicator for personal scope previews. Click the name to view details.
+- **Repository** - The repository hosting the code, with a link to view it in your source control provider
+- **Branch** - The Git branch being deployed in this preview
+- **PR** - A link to the associated pull request
+- **Owner** - The user who created the preview environment
+- **Status** - The current status of the preview (e.g., active, sleeping, deploying)
+- **Last Updated** - When the preview was last modified
+- **Actions** - Available actions for managing the preview
+
+## Managing Preview Environments
+
+Each preview has an actions menu with the following options:
+
+- **Delete** - Remove the preview environment. This option is only available if you have the necessary permissions.
+- **Wake Preview** - Wake up a sleeping preview environment to make it active again
+- **Sleep Preview** - Put an active preview to sleep to save resources. This option is disabled for previews marked as "Keep Awake"
+- **Keep Awake** - Toggle to mark a preview as persistent, preventing it from automatically sleeping due to inactivity
```

Comment:
In case it worths to mention (here or in the tip below), it also prevents its deletion